res/database/Default/database_programmes.xml

```diff
@@ -1588,15 +1668,17 @@ Erfolgreiche Forsetzung des Schreckens</de>
 		<programme id="00fd4fd6-3f9e-4231-b808-ec396f90a65d" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0050212" creator="">
 			<title>
 				<de>Die Brücke beim Kai</de>
-				<en></en>
+				<en>The Bridge by the Quay</en>
 			</title>
 			<title_original>
 				<de>Die Brücke am Kwai</de>
+				<en>The Bridge on the River Kwai</en>
 			</title_original>
 			<description>
 				<de>Während des Zweiten Weltkrieges sollen britische Kriegsgefangene für die Japaner eine Eisenbahnbrücke über einen Fluss bauen.
 Klassiker von [0|Full]</de>
-				<en></en>
+				<en>During World War II, British prisoners of war are ordered to build a railroad bridge across a river for the Japanese.
+Classic from [0|Full]</en>
```

Comment:
period; my feeling is always that "by" instead of "from" sounds better in this use case